### PR TITLE
chore(update): postcss-value-parser

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

update deps

> Is there anything in the PR that needs further explanation?

Refactor from https://github.com/stylelint/stylelint/pull/4163

Also find some bugs with new version, working on new release without bugs

### CODE REVIEW

File: lib/__tests__/printConfig.test.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ it("printConfig with multiple paths should throw", () => {
 });
 
 it("printConfig with globs should throw", () => {
-  expect(
+  return expect(
```
Comment (evilebottnawi) [inline lib/__tests__/printConfig.test.js]: After update eslint plugin jest throw errors o n this, just fix eslint problem

File: lib/utils/getUnitFromValueNode.js [hunk 2]
```diff
@@ -16,21 +15,26 @@ module.exports = function(node /*: Object*/) /*: ?string*/ {
...
+  if (!isStandardSyntaxValue(node.value)) {
+    return null;
+  }
+
+  // Ignore HEX
+  if (node.value[0] === "#") {
+    return null;
+  }
+
+  // Remove non standard stuff
```
Comment (evilebottnawi) [inline lib/utils/getUnitFromValueNode.js]: Here interesting case, for example:
```
a { margin: calc(100% - #{$margin * 2pix}); }
``` 

Now we throw error on `pix`, but it is part of interpolation (i.e. non standard) and i think we should ignore content in/into non standard syntax, but i am not sure, need some feedback

**Comment by evilebottnawi** [comment]
Some stuff from `unit` and `value` rules should be removed on refactor, i will do this in next PRs

File: lib/utils/getUnitFromValueNode.js (hunk 2 above)
Comment (jeddy3) [reply lib/utils/getUnitFromValueNode.js]: > i think we should ignore content in/into non standard syntax

Yep, I think it's best if we ignore things within interpolation. The change in behaviour will be non-breaking.

Comment (evilebottnawi) [reply lib/utils/getUnitFromValueNode.js]: @jeddy3 let's do it in other PR

**Pull request merged**